### ISSUE TITLE
pass config

### ISSUE THREAD

**Issue by ckrad** [papandreou/node-pngquant#9]
how can i pass config?
i pass ['--quality=60'] it not work

``` javascript
streamifier.createReadStream(data).pipe(new PngQuant(['--quality=60']))
```

**Comment by papandreou** [comment]
If that's a valid switch, it should work. You coulis also try `['--quality', 60]`, does that help?

**Comment by ckrad** [comment]
i tried but not work

**Comment by papandreou** [comment]
Could you try adding a test based on eg. https://github.com/papandreou/node-pngquant/blob/master/test/PngQuant.js#L17-L26 that demonstrates the problem?

**Comment by perryh** [comment]
This issue is old but if anyone is curious, here is how you can pass in configs:
```
const pngQuanter = new PngQuant(['--quality', '70-90', '-']);
```

**Comment by papandreou** [comment]
Should be addressed by #12.
